src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("    Hello from\n" + logo);
+        System.out.println("    _________________________________________");
+
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        System.out.println("    _________________________________________");
+        System.out.println("     ");
+
+        //Read in input from user
+        String inputString;
+        Scanner in;
+
+        //Set up list to store user inputs
+        Todo[] tasks = new Todo[100];
+        int counter = 0;
+
+        //setup of exit flag
+        boolean exit = false;
+        while (!exit) {
+            in = new Scanner(System.in);
+            inputString = in.nextLine();
+
+            //switch cases for all specified input types
+            switch(inputString) {
+
+            case "bye" :
+                exit = true;
+                break;
+
+            case "list" :
+                System.out.println("    _________________________________________");
+                printListContents(tasks, counter);
+                break;
+
+            case "mark" :
```

Comment:
Maybe, you could try to extract these commands as seperate methods which are then called in the main function instead of having all these commands listed out in full in the main function.

The switch statement above could also be extracted to be a seperate method which could then call these command methods.

This could improve the flow of your code.